src/Utilities/Compiler/Analyzer.Utilities.projitems

```diff
@@ -88,6 +87,7 @@
     <Compile Include="$(MSBuildThisFileDirectory)Options\Unit.cs" />
     <Compile Include="$(MSBuildThisFileDirectory)Options\ValueUsageInfo.cs" />
     <Compile Include="$(MSBuildThisFileDirectory)PathHelper.cs" />
+    <Compile Include="$(MSBuildThisFileDirectory)PerformanceSensitiveAttribute.cs" />
```

Comment:
Moved the file into this folder as it was the only place it was still being referenced.